src/evagram_input/input_tool/input_tool.py

```diff
@@ -131,22 +144,26 @@ def _add_current_experiment(self, experiment_name, owner_id):
         # and returns its identifier in the database
         self._cursor.execute("""SELECT (experiment_id) FROM experiments
                     WHERE experiment_name=%s AND owner_id=%s""", (experiment_name, owner_id))
-        current_experiment = self._cursor.fetchall()
-        if len(current_experiment) == 1:
+        current_experiment = self._cursor.fetchone()
+        if current_experiment:
             # returns experiment_id of current experiment
-            return current_experiment[0][0]
+            return current_experiment[0]
         else:
+            # generate creation date for experiment
+            timezone = pytz.timezone("US/Eastern")
+            create_date = datetime.now(timezone)
```

Comment:
Thanks for catching this, I was going to ask about this but now the idea is to convert all the timestamps to UTC time? It also appears that `datetime.utcnow()` is deprecated since version 3.12 so I will use `datetime.now(timezone.utc)`.

Reply:
Yeah for now convert timestamps to UTC